documentation/examples/hello_cli/manifest.ts

```diff
@@ -0,0 +1,13 @@
+import type {PackVersionDefinition} from '@codahq/packs-sdk';
+import {formats} from './formulas';
+import {formulas} from './formulas';
+import {syncTables} from './formulas';
+
+export const manifest: PackVersionDefinition = {
```

Comment:
Just curious on your opinion thus far of the newPack() builder pattern, and whether we should use that for the CLI as well? No need to change anything now, just raising the question.

Reply:
I've been concerned about how the Pack Editor and CLI use two different "dialects", and the inability to easily copy and paste between them. I'll start a discussion in Slack to explore this more.